### PR TITLE
DOCS-4119 SF ref update

### PR DESCRIPTION
**Pull request by george-hoffer**
https://www.mulesoft.org/jira/browse/DOCS-4119

### CODE REVIEW

File: modules/ROOT/pages/salesforce/salesforce-connector-reference.adoc [hunk 1]
```diff
@@ -1,3029 +1,4499 @@
...
-| Authorization URL a| String |Web service URL responsible for user authentication. This is the URL for the endpoint that is configured to handle SOAP authentication requests. |`+https://login.salesforce.com/services/Soap/u/41.0+` |
-| Reconnection a| Reconnection |When the application is deployed, a connectivity test is performed on all connectors. If set to true, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy. | |
-| Pooling Profile a| PoolingProfile |Characteristics of the connection pool | |
-| Read Timeout a| Number |Specifies the amount of time, in milliseconds, that the consumer waits for a response before it times out. Default value is 0, which means infinite. |0 |
-| Connection Timeout a| Number |Specifies the amount of time, in milliseconds, that the consumer attempts to establish a connection before it times out. Default value is 0, which means infinite. |0 |
-| Assignment Rule Id a| String | | |
-| Client Id a| String |Client ID for partners. | |
-| Time Object Store a| ObjectStore |An ObjectStore instance to use in SalesforceExtension#getUpdatedObjects. | |
```
Comment (wwarner-salesforce) [inline modules/ROOT/pages/salesforce/salesforce-connector-reference.adoc:43]: This was changed from "An ObjectStore instance to use in SalesforceExtension#getUpdatedObjects." to "An ObjectStore instance to use in SalesforceExtension.". I wonder if it meant that the instance should be used in an extension called getUpdatedObjects, and if the sentence should read "An ObjectStore instance to use in the SalesforceExtension getUpdatedObjects."

Comment (wwarner-salesforce) [reply modules/ROOT/pages/salesforce/salesforce-connector-reference.adoc:43]: (Here I'm comparing to line 55 of the new file; lines 87 and 113 still have "SalesforceExtension#getUpdatedObjects".)

Comment (george-hoffer) [reply modules/ROOT/pages/salesforce/salesforce-connector-reference.adoc:43]: The original entry was:
| Time Object Store a| <<ObjectStore>> |  +++An ObjectStore instance to use in <a href="#_get_updated_objects">SalesforceExtension#getUpdatedObjects</a>
However there's no [[ObjectStore]] or [[_get_updated_objects]] anchors in this file for these links to link to. Similarly the Get Updated Objects operation has no ObjectStore reference. 

Changing this to:
An ObjectStore instance.

File: modules/ROOT/pages/salesforce/salesforce-connector-reference.adoc [hunk 2]
```diff
@@ -3401,38 +5014,38 @@ Subscribe to a topic. First the topic must be published and after that a subscri
...
-| Search Layoutable a| Boolean | | |
-| Searchable a| Boolean | | |
-| Triggerable a| Boolean | | |
-| Undeletable a| Boolean | | |
-| Updateable a| Boolean | | |
-| Url Detail a| String | | |
-| Url Edit a| String | | |
-| Url New a| String | | |
+| Action Overrides a| Array of <<ActionOverride>> |  |  | 
+| Activateble a| Boolean |  |  |
```
Comment (wwarner-salesforce) [inline modules/ROOT/pages/salesforce/salesforce-connector-reference.adoc:5018]: Wanted to double check this change from "Activateable" to "Activateble"

**Review by wwarner-salesforce — APPROVED** [review]
Looks good! I didn't build to ensure there are no build errors and the formatting renders correctly, but I assume that you have. Due to the length I wasn't able to read the whole PR in close detail, but I skimmed pretty closely and spot-checked, and it all looks good to me.

File: modules/ROOT/pages/salesforce/salesforce-connector-reference.adoc (hunk 2 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/salesforce/salesforce-connector-reference.adoc:5018]: George confirmed this.

**Pull request merged**